lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -65,8 +68,11 @@
                         },
                         error: function () {
                             view.showNotificationMessage(
-                                view.indicators.error +
-                                gettext('You must sign out and sign back in before your language changes take effect.')
+                                HtmlUtils.joinHtml(
+                                    view.indicators.error,
+                                    gettext('You must sign out and sign back in ' +
+                                        'before your language changes take effect.')
```

Comment:
This sentence needs to remain all on one line to ensure correct translation. If that's causing line-too-long quality failures, you can make an exception in jshint like this: https://github.com/edx/edx-platform/blob/master/lms/static/js/certificates/views/certificate_invalidation_view.js#L49